### PR TITLE
Adding some docs for the selectable interface

### PR DESCRIPTION
**Pull request by apaganobeleno**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,6 +237,22 @@ Which will use the same value for the `value` attribute and the body of the opti
 
 Which (given the Plush power) allows us to define the options map inside the view.
 
+#### Selectable Interface
+
+Another alternative for the select options is to pass a list of structs that meet `form.Selectable` interface.
+
+Which consist in two functions:
```
Comment (markbates) [inline README.md]: Which consist of two functions

File: README.md [hunk 2]
```diff
@@ -237,6 +237,22 @@ Which will use the same value for the `value` attribute and the body of the opti
 
 Which (given the Plush power) allows us to define the options map inside the view.
 
+#### Selectable Interface
+
+Another alternative for the select options is to pass a list of structs that meet `form.Selectable` interface.
```
Comment (markbates) [inline README.md]: that meet the `form.Selectable` interface

File: README.md [hunk 3]
```diff
@@ -237,6 +237,22 @@ Which will use the same value for the `value` attribute and the body of the opti
...
+
+```go
+//Selectable allows any struct to become an option in the select tag.
+type Selectable interface {
+	SelectValue() interface{}
+	SelectLabel() string
+}
+```
+
+By implementing this interface tags will call `SelectValue` and `SelectLabel` to get the option Value and Label from the struct instance.
```
Comment (markbates) [inline README.md]: By implementing this interface `tags` will call `SelectValue` and `SelectLabel` to get the option Value and Label from implementer.

**Comment by apaganobeleno** [comment]
Thanks @markbates !

**Pull request merged**